README.md

```diff
@@ -115,6 +115,11 @@ if err != nil {
 log.Print(res) // do something with the result
 ```
 
+#### Launchpad usage
```

Comment:
`Launchpad usage` -> `Launchpad Usage` to follow the style of the rest of the repo